pyanaconda/modules/storage/devicetree/handler_interface.py

```diff
@@ -46,14 +46,15 @@ def TeardownDevice(self, device_name: Str):
         """
         self.implementation.teardown_device(device_name)
 
-    def MountDevice(self, device_name: Str, mount_point: Str):
+    def MountDevice(self, device_name: Str, mount_point: Str, options: Str):
         """Mount a filesystem on the device.
 
         :param device_name: a name of the device
         :param mount_point: a path to the mount point
+        :param options: options used for device mount, could be an empty string
```

Comment:
Please, specify it as `a string with mount options or an empty string to use defaults`. Same in the handler.